src/standalone/topbar/topbar.jsx

```diff
@@ -387,6 +387,14 @@ export default class Topbar extends React.Component {
               { this.state.clients
                   .map((cli, i) => <li key={i}><button type="button" onClick={() => this.downloadGeneratedFile("client", cli)}>{cli}</button></li>) }
             </DropdownMenu> : null }
+            <DropdownMenu {...makeMenuOptions("About")}>
+              <li><a href="https://swagger.io/tools/swagger-editor/" target="_blank" rel="noreferrer noopener">About Swagger Editor</a></li>
+              <li><a href="https://swagger.io/docs/open-source-tools/swagger-editor/" target="_blank" rel="noreferrer noopener">View Docs</a></li>
+              <li><a href="https://github.com/swagger-api/swagger-editor" target="_blank" rel="noreferrer noopener">View on GitHub</a></li>
+            </DropdownMenu>
+            <Link href="https://editor-next.swagger.io/" className="new-editor-cta" target="_blank" rel="noreferrer noopener">
```

Comment:
`rel="noreferrer noopener"` attribute is added automatically by the `<Link>` component.

```suggestion
            <Link href="https://editor-next.swagger.io/" className="new-editor-cta" target="_blank">
```